src/style.css

```diff
@@ -1,3 +1,132 @@
-body {
+body,
+h2,
+p {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: start;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  padding-top: 33px;
+}
+
+.card__about {
+  display: flex;
+  padding: 40px 0 16px;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.card__title {
+  line-height: 18px;
+  color: #060b35;
```

Comment:
move repeated colors to varaibles

Reply:
Still not fixed from the previous review